app/api.py

```diff
@@ -38,3 +44,194 @@ def main_page(request: Request):
 @app.get("/robots.txt", response_class=PlainTextResponse)
 def robots_txt():
     return """User-agent: *\nDisallow: /"""
+
+
+# CRUD Flags
+
+
+class TicketCreate(BaseModel):
+    barcode: str = Field(..., description="Barcode of the product")
+    type: str = Field(..., description="Type of the issue")
+    url: str = Field(..., description="URL of the product")
+    status: str = Field(..., description="Status of the ticket")
+    image_id: str = Field(..., description="Image ID of the product")
+    flavour: Flavor = Field(..., description="Flavour of the product")
+    created_at: datetime = Field(default_factory=datetime.utcnow)
+
+
+class Ticket(TicketCreate):
+    id: int = Field(..., description="ID of the ticket")
+
+
+class FlagCreate(BaseModel):
+    barcode: str = Field(..., description="Barcode of the product")
+    type: str = Field(..., description="Type of the issue")
+    url: str = Field(..., description="URL of the product")
+    user_id: str = Field(..., description="User ID of the flagger")
+    device_id: str = Field(..., description="Device ID of the flagger")
+    source: str = Field(..., description="Source of the flag")
+    confidence: float = Field(..., description="Confidence of the flag")
```

Comment:
You can add it's only meant for Robotoff predictions (machine learning confidence score). And it's an optional param!